### DISCUSSION TITLE
[x][postponed][manuscript][outreach] Interoperability with UCSC formats?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#111] [Team Posts]
What are the issues. How much will we be able to do?

**Comment by dzerbino** [comment]
Email sent to UCSC. On principle they are fine with it, but Jim Kent cannot respond at the moment.

**Comment by sveinugu** [comment]
Had what evolved into a reliatively long mail exchange with a contact at UCSC early December (2019), after Daniel introduced me and other parties interested in JSON support as input to UCSC. No concrete output, but at least there is a thread to pick up on. This was basically about creating a JSON Schema of the existing UCSC TrackHub format, and supporting import of that on the UCSC side. I also raise the question about being able to host a trackhub via an API, and I was encouraged to open up a helpdesk request for this, which I have forgotten to do.

**Comment by sveinugu** [comment]
Postponing, as we are closing shop.

**Comment by sveinugu** [comment]
Added links to this from https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/41 and https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/49
